### DISCUSSION TITLE
Can the new jobs be appended at the top rather than sorting by date?

### DISCUSSION THREAD

**Discussion by HarshAgarwal2703** [ReaVNaiL/New-Grad-2024#112] [Ideas]
Hello,
Can we have the jobs which are getting updated or added be appended to the top of the list rather than sorting them by date? Sometimes there is a chance of missing a new posting as it was down in the list, and we have to check if we have applied to it or not. Appending the jobs being added to the top will make it easier to look for them and apply for them.
Thank you!

**Comment by ReaVNaiL** [comment]
I will update the workflow and the structure to help with this!
